build.sh

```diff
@@ -15,24 +15,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -e                # exit on error
+set -xe
```

Comment:
No need for the `-x` before the `usage()`.

Reply:
I don't disagree, but I might go further and suggest we don't need -x on by default at all. :)